### DISCUSSION TITLE
sqlalchemy makes me giggle sometimes

### DISCUSSION THREAD

**Discussion by nexushoratio** [sqlalchemy/sqlalchemy#11951] [Usage Questions]
Just saw this in a stack trace:
```
to_update, replacement_expr = leep(
```

And, sometimes I notice this in my logs:
```
base.py:750(_finalize_fairy)]
```

Not sure what `leep` means in this context (and it doesn't matter), but just seeing it made me smile, as does the fairy.

So, just a thanks for injecting a tad more humor into development!

**Comment by CaselIT** [comment]
Seems to be a simple abbreviation :) https://github.com/sqlalchemy/sqlalchemy/blob/fe03298593e59e5d40de81e7f864690e73930666/lib/sqlalchemy/sql/compiler.py#L2099-L2102

**Reply by nexushoratio** [reply]
Well, yes.  But it was more fun to imagine a lemur hopping around the screen chanting "leep, leep, leep" (reference [Zoboomafoo](https://en.wikipedia.org/wiki/Zoboomafoo)).  Then again, I probably should neither write code nor post when somewhat sick.
